src/Native/configure.cmake

```diff
@@ -270,6 +270,13 @@ check_function_exists(
     inotify_rm_watch
     HAVE_INOTIFY_RM_WATCH)
 
+check_cxx_source_compiles(
+    "
+    #include <curl/multi.h>
+    int main() { int i = CURLM_ADDED_ALREADY; }
+    "
+    HAVE_CURLM_ADDED_ALREADY)
+
 set (HAVE_INOTIFY 0)
```

Comment:
(minor) Setting the HAVE_INOTIFY variable uses the 3 variables directly above it (HAVE_INOTIFY_RM_WATCH, et al). I think the new check for CURLM_ADDED_ALREADY should go after this, to keep the HAVE_INOTIFY* stuff together.